packages/units/src/getBoundRect.js

```diff
@@ -130,6 +131,16 @@ export function getCorners(angle, x, y, width, height) {
   };
 }
 
+export function isElementBelowLimit(element, verifyLink = true) {
```

Comment:
This isn't really related to the units nor with `getBoundRect` -- what's the reason for moving this to the units package?

Reply:
Sounds like something that might depend on the `getBoundRect` but not something that one would intuitively look for from the units package. This is directly related to the editor only, why not just keep it under the utils there? Thoughts? Or what kind of functionality is the unit packages intended to include?

Reply:
Used by output package perhaps? Or used to be

Reply:
@swissspidy Yep, that's true but I assumed the output generation also belongs to the editor, is there a plan to move the output to a separate package, too?

Reply:
Perhaps in the future there could be a package for utils related to links (including page attachments) where the `isElementBelowLimit` could belong. As it's currently used only in the `edit-story` then I think it would make sense to just move it back to the `utils` folder. `isElementBelowLimit` is directly related to the behaviour of links only and shouldn't be under the `units` package IMO.